### DISCUSSION TITLE
Win11, Windows 10 File Explorer, Bring back downloads, documents, 3d objects, etc on the left side bar.

### DISCUSSION THREAD

**Discussion by UMesssedUp** [valinet/ExplorerPatcher#3265] [Q&A]
I am happy to have the windows 10 file explorer back on windows 11 but the same primary issue with both file explorers I have is that downloads and documents and all the others aren't under "This PC" and they're only as quick access pins, but quick access pins cant be hovered over and dropped down to see folders inside.
![image](https://github.com/valinet/ExplorerPatcher/assets/81893479/ae752cef-5123-4d51-b6e5-3fa546fc33a4)
(Not a great image but you see what I mean hopefully)
![9898c0a8-8d64-46e7-b572-0583c645af7e](https://github.com/valinet/ExplorerPatcher/assets/81893479/0654a5f3-d037-4f4f-91ca-d1f3dec2f776)
they are all there and all have the drop down triangle which automatically opens up when a file or folder is dragged over it.

**Comment by pyrates999** [comment]
https://support.microsoft.com/en-us/windows/how-to-get-to-the-desktop-documents-and-pictures-folders-in-file-explorer-3370f06b-0f8d-4b25-be9a-3ee54f381e3d

**Reply by UMesssedUp** [reply]
This doesn't make them appear under "This PC" like on windows 10.

**Reply by pyrates999** [reply]
what does it do then?

**Reply by UMesssedUp** [reply]
that just puts them in quick access which is useless for me since I cant drag files over downloads for example and have it open up to see "tools" or "Text Files" folders to put a file in there rather than dragging it to downloads, opening downloads quick access pin, and then dragging it into the folder.

**Reply by pyrates999** [reply]
Did you try the section "Restore the left navigation pane if you have hidden it"?

**Reply by UMesssedUp** [reply]
Yes I have and they don't change it. I had to set it back to windows 11 explorer to do it and it didnt bring them back under This PC

**Reply by pyrates999** [reply]
Alright, please wait for a developer to respond.
